### PR TITLE
Support mas (Mac App Store) platform

### PR DESCRIPTION
**Pull request by aluxian**

### CODE REVIEW

File: plugin.js [hunk 1]
```diff
@@ -25,8 +25,8 @@ function optionDefaults (options, callback) {
         check we're on the right platform, before we go any further
     ****** */
 
-    if (['darwin','linux','win32'].indexOf(options.platform) < 0) {
-        return cb(new Error('Only darwin, linux or win32 platforms are supported'));
+    if (['darwin','mas','linux','win32',].indexOf(options.platform) < 0) {
+        return cb(new Error('Only darwin, mas, linux or win32 platforms are supported'));
     }
 
     if (options.platform === 'darwin' && options.arch !== 'x64') {
```
Comment (smebberson) [inline plugin.js]: I assume the `mas` platform only supports `x64` architecture too? Does this line need to take that into consideration?

**Comment by smebberson** [comment]
@Aluxian, thanks for the PR. I just wrote an inline comment. Are you able to respond?

**Comment by aluxian** [comment]
TBH I wasn't expecting such a fast reply, haha. I'll tidy up this PR.

**Comment by aluxian** [comment]
Done. I updated the README, too. Anything else to be done before merging?

**Pull request merged**